***На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?***
---

Формула Z = (x - mu) / sigma 

sigma = корень из дисперсии, т.е. корень из 25

mu - среднее (математическое ожидание) = 178

x = 190 

In [3]:
from math import sqrt

def Z(x, mu, sigma):
    return (x - mu) / sigma


sigma = sqrt(25)

Z(190, 178, sigma)

2.4

Ответ: На 2.4 сигмы отклоняется рост человека, равный 190 см